test/integration/bench_memif_test.go

```diff
@@ -0,0 +1,173 @@
+package nsmd_integration_tests
```

Comment:
You need to have this line in the beginning of the file to make it run during the basic tests phase:
```golang
// +build basic
```

Reply:
I just testing memif_test on CI, I've marked this PR as WIP. I'll review all comments later

Reply:
Fixed